server/resolvers/mutation.ts

```diff
@@ -116,39 +110,31 @@ export const Mutation = prismaObjectType({
         };
 
         const hashPassword = ({ password }: Schema) =>
-          tryCatch(() => hash(password), constant('UNREACHABLE ID: 5'));
+          TE.tryCatch(constant(hash(password)), constant('UNREACHABLE ID: 5'));
```

Comment:
i don't think this should be refactored into using constant, as it's a lot more confusing. this isn't a constant, it's an `IO`. it only type checks here due to the way `IO` is implemented. I think you also realised this and changed other occurences of this in 2b0ab1ae0a982c6324303dcc5d2f94d03a89c06b but this one slipped through the cracks.